python/ray/serve/_private/deployment_graph_build.py

```diff
@@ -145,7 +146,11 @@ def transform_ray_dag_to_serve_dag(
                     DeploymentFunctionNode,
                 ),
             ),
-            apply_fn=lambda node: DeploymentHandle(node._deployment.name, app_name),
+            apply_fn=lambda node: DeploymentHandle(
+                node._deployment.name,
+                app_name,
+                handle_options=_HandleOptions(_component=HandleComponent.REPLICA),
```

Comment:
reading this usage makes me want it to be named `_source` even more :)

Reply:
haha sg, changed